test/api/core/hotelClerk/removeCustomerFromAllRooms.test.js

```diff
@@ -64,7 +64,7 @@ describe('Test: hotelClerk.removeCustomerFromAllRooms', () => {
     hotelClerk.removeCustomerFromAllRooms(context);
 
     should(kuzzle.realtime.remove).be.calledOnce();
-    should(kuzzle.notifier.notifyUser).be.calledOnce();
+    should(kuzzle.notifier.notifyUser).be.calledTwice();
```

Comment:
niptickings:
* Add a comment to explain why the `notifyUser` method shoud be called twice
* The test title is not about notifications, only about cleanup: to be more "unit", the call to "notifyUser" should be tested in another test.

Reply:
- about the comment: I disagree. From the test perspective, the customer is registered to 2 rooms. IMO, it is more natural that we send 2 notifications than one. There should have been a comment on the original version though.
- OK, I split the tests